benchmarks/benchmark/tools/profile-generator/container/latency_throughput_curve.sh

```diff
@@ -25,7 +25,7 @@ for request_rate in $(echo $REQUEST_RATES | tr ',' ' '); do
   # TODO: Check if profile already exists, if so then skip
   timestamp=$(date +"%Y-%m-%d_%H-%M-%S")
   output_file="latency-profile-${timestamp}.txt"
-  PYTHON_OPTS="$PYTHON_OPTS --host=$IP   --port=$PORT   --model=$TOKENIZER --dataset=ShareGPT_V3_unfiltered_cleaned_split.json --tokenizer=$TOKENIZER --request-rate=$request_rate --backend=$BACKEND --num-prompts=$((request_rate * 30)) --max-input-length=$INPUT_LENGTH --max-output-length=$OUTPUT_LENGTH"
+  PYTHON_OPTS="$PYTHON_OPTS --save-json-results --host=$IP   --port=$PORT   --model=$TOKENIZER --dataset=ShareGPT_V3_unfiltered_cleaned_split.json --tokenizer=$TOKENIZER --request-rate=$request_rate --backend=$BACKEND --num-prompts=$((request_rate * 30)) --max-input-length=$INPUT_LENGTH --max-output-length=$OUTPUT_LENGTH"
```

Comment:
Not a blocker for this PR, but wondering if we need both the "save-json-results" and the > "output.txt" ? do they accomplish the same thing?

Reply:
Yes and no, same data different format

Example .txt output:
```
Namespace(backend='jetstream', sax_model='', endpoint='generate', host='jetstream-svc', port=8000, dataset='ShareGPT_V3_unfiltered_cleaned_split.json', model='google/gemma-7b', tokenizer='google/gemma-7b', best_of=1, use_beam_search=False, num_prompts=300, max_input_length=256, max_output_length=256, top_k=32000, request_rate=10.0, seed=0, trust_remote_code=False, machine_cost=None, use_dummy_text=False, save_json_results=True, additional_metadata_metrics_to_save=None, scrape_server_metrics=False)
Total time: 34.68 s
Requests/min: 519.10
Output_tokens/min: 49312.80
Input_tokens/min: 35499.54
Tokens/min: 84812.33
Average seconds/request (includes waiting time on server): 2.37
Average milliseconds/token (includes waiting time on server): 15.67
Average milliseconds/output_token (includes waiting time on server): 52.73
Average input length: 68.39
Average output length: 95.00
```

Example .json output:
```
{"metrics": {"num_prompts": 450, "request_rate": 15.0, "benchmark_time": 39.20198845863342, "total_output_token": 43164, "output_tokens_per_min": 66063.99577748055, "total_input_tokens": 31031, "input_tokens_per_min": 47494.019390487425, "total_tokens": 74195, "tokens_per_min": 113558.01516796798, "avg_latency": 5.056885840627882, "avg_per_token_latency": 0.03982198388778007, "avg_per_output_token_latency": 0.3884640749165082, "avg_input_len": 68.95777777777778, "avg_output_len": 95.92}, "dimensions": {"date": "20240913-164005", "backend": "jetstream", "model_id": "google/gemma-7b", "tokenizer_id": "google/gemma-7b"}}
```

Reply:
Ah from what you posted, the output.txt is a bit more readable than the json :) so likely still useful for people who don't want to reuse the json for something else.